src/lammpsio/box.py

```diff
@@ -116,6 +149,85 @@ def from_matrix(cls, low, matrix, force_triclinic=False):
 
         return cls(low, high, tilt)
 
+    def to_hoomd_convention(self):
+        """Convert a :class:`Box` to HOOMD-blue convention.
+
+        Parameters
```

Comment:
This docstring should at least link to the HOOMD docs on what this convention is.